config/projects.yaml

```diff
@@ -415,10 +415,10 @@ timeline-bundle:
 translation-bundle:
     branches:
         master:
-            php: ['7.2', '7.3', '7.4']
+            php: ['7.3', '7.4']
             variants:
                 symfony/symfony: ['4.4']
-                sonata-project/admin-bundle: ['3']
+                sonata-project/admin-bundle: ['4.x-dev']
```

Comment:
better use dev-master here, to do the same than on doctrine orm admin

Reply:
why is it better? just out of curiosity

Reply:
just consistency and ensure the twig filter that @VincentLanglet implemented works the same way

Reply:
Cf: https://github.com/sonata-project/dev-kit/blob/master/src/Twig/Extension/GithubExtension.php#L19
And https://github.com/sonata-project/dev-kit/blob/master/templates/project/.github/workflows/test.yaml.twig#L53

But maybe there is something better to do ? 

Without any change, we will try to run test with the version `'4.x-dev.x'`